src/api/app/jobs/project/update_packages_if_dirty.rb

```diff
@@ -0,0 +1,7 @@
+class Project
```

Comment:
Nested where? what is the namespace for it?
What if I come after you and I add something like this in this file:
```ruby
class Project

  def name
    "whatever"
  end
.... your actual code
end
```

Then I'm overriding the **name** method of the Project model